### PR TITLE
Set monthly savings goal

### LINKED ISSUE #35: Set monthly saving goals
As a user, I want to set monthly saving goals so that I can cultivate good saving habits.

### PR DESCRIPTION
**Pull request by zhaojj2209**
Resolves #35.

Added `SetMonthlySavingsGoalCommand` to allow users to set their monthly savings goal in the finance tracker.

### CODE REVIEW

File: src/main/java/ay2021s1_cs2103_w16_3/finesse/model/FinanceTracker.java [hunk 1]
```diff
@@ -151,10 +151,16 @@ public void setExpenseLimit(Amount limit) {
...
+    public void setSavingsGoal(Amount goal) {
+        monthlyBudget.setMonthlySavingsGoal(goal);
+    }
+
     public void setMonthlyBudget(MonthlyBudget budget) {
         monthlyBudget.setMonthlyExpenseLimit(budget.getMonthlyExpenseLimit().getAmount());
+        monthlyBudget.setMonthlySavingsGoal(budget.getMonthlySavingsGoal().getAmount());
     }
 
+
```
Comment (ianyong) [inline src/main/java/ay2021s1_cs2103_w16_3/finesse/model/FinanceTracker.java:163]: Remove this?

File: src/main/java/ay2021s1_cs2103_w16_3/finesse/model/budget/MonthlyBudget.java [hunk 2]
```diff
@@ -7,9 +7,14 @@
  */
 public class MonthlyBudget {
     private MonthlyExpenseLimit monthlyExpenseLimit;
+    private MonthlySavingsGoal monthlySavingsGoal;
 
+    /**
+     * Creates a {@code MonthlyBudget}.
```
Comment (ianyong) [inline src/main/java/ay2021s1_cs2103_w16_3/finesse/model/budget/MonthlyBudget.java:13]:
```suggestion
     * Creates a {@code MonthlyBudget} with an expense limit and savings goal of $0.
```

Comment is not descriptive.

**Review by ianyong — APPROVED** [review]
Other than a few nits, LGTM!

**Review by ianyong — APPROVED** [review]
LGTM!

**Review by siddarth2824 — APPROVED** [review]
LGTM!

**Pull request merged**